jsdoc/processors/inline-tags.js

```diff
@@ -1,5 +1,5 @@
 var _ = require('lodash');
-var INLINE_TAG = /\{@([^\s]+)\s+([^\}]*)\}/g;
+var INLINE_TAG = /(?:^|)\{@([^\s]+)\s+(.*)\}(?:$|)/g;
```

Comment:
I am not much into dgeni (yeah, shame on me), but wouldn't this incorectly match until the last inline tag's closing `}` ?